.env.test

```
@@ -3,4 +3,6 @@ ENDPOINT_PORT=6381
 ENDPOINT_HOST=localhost
 EMAIL_PUBLIC_URL=http://localhost:3000
 EMAIL_TRANSPORT="smtp://localhost:1025"
-EMAIL_FROM="test@revisium.local"
\ No newline at end of file
+EMAIL_FROM="test@revisium.local"
+FILE_PLUGIN_PUBLIC_ENDPOINT=http://test-files.localhost
+FILE_PLUGIN_PUBLIC_ENDPOINT=http://test-files.localhost
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The FILE_PLUGIN_PUBLIC_ENDPOINT key is duplicated

(DuplicatedKey)